src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -38,6 +38,32 @@ class AnalyticsConstants {
         }
     }
 
+    public static def additional_multiscm = { allVars ->
```

Comment:
Are these used in any other job?  If not, maybe this should be defined in EventExportIncremental.groovy